### PR TITLE
PR 17/01/2024

### PR DESCRIPTION
**Pull request by BabelDev0**

### CODE REVIEW

File: content/QCM Physics/07_Quantum Hall effect/attachments/cyclotron.png
Comment (deskclam) [inline content/QCM Physics/07_Quantum Hall effect/attachments/cyclotron.png:1]: wrong sign as the charge is negative, so after the right hand rule the force points in the opposite direction: F=qv x B =-ev x B

**Review by deskclam — CHANGES_REQUESTED** [review]
I pointedo out just some errors or things to change a bit

File: content/QCM Physics/07_Quantum Hall effect/attachments/cyclotron.png
Comment (BabelDev0) [reply content/QCM Physics/07_Quantum Hall effect/attachments/cyclotron.png:1]: I think that for negative particle we must use the left-hand rule (thumb in the direction of the particle's velocity, your index finger in the direction of the magnetic field, and your middle finger will point in the direction of the force on the particle) @deskclam

File: content/QCM Physics/05_2DEG/2DEG.md
[the remark this answers is not in the document]
Comment (BabelDev0) [reply content/QCM Physics/05_2DEG/2DEG.md:1]: mmh I can't figure out what the error is: maybe try to come up with a corrected version of the sentence. @deskclam

File: content/QCM Physics/07_Quantum Hall effect/attachments/cyclotron.png
Comment (deskclam) [reply content/QCM Physics/07_Quantum Hall effect/attachments/cyclotron.png:1]: ok is the same left hand or right hand and sign change but in any case I was wrong and the image is correct

**Pull request merged**